### PR TITLE
Add publish-kedro-viz action to Kedro-Viz docs. 

### PR DESCRIPTION
**Pull request by rashidakanchwala**
## Description

Add `publish-kedro-viz` action to Kedro-Viz docs. 

## Checklist

- [ ] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: docs/source/share_kedro_viz.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ The publish and share feature on Kedro-Viz enables seamless sharing of pipeline
 * [Publish and share on Azure](./publish_and_share_kedro_viz_on_azure)
 * [Publish and share on GCP](./publish_and_share_kedro_viz_on_gcp)
 
+3. **Publish and Share Kedro-Viz with Github Actions**: Use the `publish-kedro-viz` action available on the Github Marketplace to deploy Kedro-Viz for your Kedro project repository on GitPages. For further information and usage instructions, please refer to [publish-kedro-viz](https://github.com/marketplace/actions/publish-kedro-viz)
```
Comment (ravi-kumar-pilla) [inline docs/source/share_kedro_viz.md:12]:
```suggestion
3. **Publish and Share Kedro-Viz with Github Actions**: Use the `publish-kedro-viz` action available on the Github Marketplace to deploy Kedro-Viz for your Kedro project repository on GitHub Pages. For further information and usage instructions, please refer to [publish-kedro-viz](https://github.com/marketplace/actions/publish-kedro-viz)
```

**Review by ravi-kumar-pilla — APPROVED** [review]
Minor suggestion. Looks good otherwise. Thanks Rashida 👍

**Comment by astrojuanlu** [comment]
Rendered result 🤖 https://kedro--1856.org.readthedocs.build/projects/kedro-viz/en/1856/share_kedro_viz.html

**Review by astrojuanlu — APPROVED** [review]
LGTM!

**Pull request merged**